src/main/java/uk/gov/justice/probation/courtcasematcher/model/domain/Hearing.java

```diff
@@ -32,6 +32,8 @@ public class Hearing implements Serializable {
 
     private final DataSource source;
 
+    private final String hearingEventType;
```

Comment:
I can't see where this is ever populated? There should be changes to the MessageProcessor to get the update type out of the message metadata but I can't see it?

Reply:
It looks like this hasn't been passed in from CHER, sorry should have spotted that.